.github/workflows/python-package.yml

```diff
@@ -23,6 +23,8 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v3
       - uses: actions/setup-python@v5
+        with:
+          cache: 'pip'
```

Comment:
`uv` cache here too (see previous comment).

Reply:
Fixed - Thank you!!